README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.org/freight-team/freight.svg?branch=master)](https://travis-ci.org/freight-team/freight)
 
```

Comment:
I had it on one line, but liked it better this way because of the different size... or maybe have the IRC button first because it's larger?